image/tarexport/load.go

```diff
@@ -129,6 +129,11 @@ func (l *tarexporter) Load(inTar io.ReadCloser, outStream io.Writer, quiet bool)
 		imageIDsStr += fmt.Sprintf("Loaded image ID: %s\n", imgID)
 
 		imageRefCount = 0
+		// add a tag from CLI like the last tag for manifest
+		// TODO: but it's working only for one image in *.tgz file
```

Comment:
I think `docker load --format "{{json .}}" -i myapp.v1.tgz | some-external-script` is better
https://github.com/moby/moby/issues/40188#issuecomment-623568574